### DISCUSSION TITLE
Wrong partner api endpoint on the docs

### DISCUSSION THREAD

**Discussion by leocb** [immich-app/immich#9763] [Q&A] [answered]
### The bug

the immich docs says it's `/api/partners` when in fact it what works for me is `/api/partner` (without the `s` at the end).
I tested only with GET, but I believe this is true for all 4 partner endpoints.

### The OS that Immich Server is running on

Ubuntu 22.10

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

n/a

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

```bash
n/a
```

**Comment by mmomjian** [comment, marked as answer]
The docs currently reflect the dev branch, which means that those API endpoints will be updated in the next release of Immich. To explore the API of your current version, you can visit http://immichip:port/doc

**Reply by leocb** [reply]
Got it! May I suggest a version label somewhere in the online API pages then? just an assurance that people are looking at the correct info.

**Reply by mmomjian** [reply]
This is being tracked in #9642
